spec/models/mention_spec.rb

```diff
@@ -25,6 +24,25 @@
     end
   end
 
+  describe "#mention_syntax" do
+    before do
+      @user = alice
+      @aspect1 = @user.aspects.create(:name => 'second_aspect')
+    end
+
+    it 'should not try to build a mention for broken syntax' do
+      sm = @user.build_post(:status_message,
+                            :text => "look at @{...} and @{blalbubb} and @{stuff; badhandle} and @{bad ;semicolon} - this is wrong syntax", :to => @user.aspects.first)
+      expect(sm.mentioned_people).to be_empty
+    end
+
+    it 'should recognize correctly mentioned people' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.